### PR TITLE
Update broken link

### PR DESCRIPTION
**Pull request by ThatStoney**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -42,4 +42,4 @@
 [1]: https://github.com/github/backup-utils/releases
 [2]: https://github.com/github/backup-utils/releases/tag/v2.11.4
 [3]: https://github.com/github/enterprise-backup-site/blob/master/backup.config-example
-[4]: https://enterprise.github.com/help/articles/adding-an-ssh-key-for-shell-access
+[4]: https://docs.github.com/en/enterprise-server@3.6/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent
```
Comment (djdefi) [inline docs/getting-started.md:45]:
```suggestion
[4]: https://docs.github.com/enterprise-server/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent
```

**Review by djdefi — COMMENTED** [review]
One minor bit on the link format

File: docs/getting-started.md (hunk 1 above)
Comment (ThatStoney) [reply docs/getting-started.md:45]: Ah, thanks forgot to pull that 👍

**Review by djdefi — APPROVED** [review]

File: docs/getting-started.md [hunk 2]
```diff
@@ -42,4 +42,4 @@
 [1]: https://github.com/github/backup-utils/releases
 [2]: https://github.com/github/backup-utils/releases/tag/v2.11.4
```
Comment (Hamidking51) [inline docs/getting-started.md:43]: usaa58358@gmail.com

**Review by Hamidking51 — COMMENTED** [review]
furnch.hmd@gmail.com

**Pull request merged**